install.sh

```diff
@@ -1300,33 +1553,39 @@ setup_shared_hook_repositories() {
 
     echo "OK, let's input them one-by-one and leave the input empty to stop."
 
-    SHARED_REPOS_LIST=""
+    PROVIDED="false"
     while true; do
         printf "Enter the clone URL of a shared repository: "
         read -r SHARED_REPO </dev/tty
         if [ -z "$SHARED_REPO" ]; then break; fi
 
-        if [ -n "$SHARED_REPOS_LIST" ]; then
-            SHARED_REPOS_LIST="${SHARED_REPOS_LIST},${SHARED_REPO}"
-        else
-            SHARED_REPOS_LIST="$SHARED_REPO"
+        if [ -n "$SHARED_REPO" ]; then
+
+            if [ "$PROVIDED" = "false" ]; then
+                git config --global --unset-all githooks.shared
+            fi
+
+            if git config --global --add githooks.shared "$SHARED_REPO"; then
+                PROVIDED="true"
+            else
+                PROVIDED="error"
+                break
+            fi
         fi
     done
 
-    if [ -z "$SHARED_REPOS_LIST" ] && git config --global --unset githooks.shared; then
+    if [ "$PROVIDED" = "false" ] &&
+        git config --global --unset githooks.shared; then
         echo "Shared hook repositories are now unset. If you want to set them up again in the future, run this script again, or change the 'githooks.shared' Git config variable manually."
         echo "Note: shared hook repos listed in the .githooks/.shared file will still be executed"
-    elif git config --global githooks.shared "$SHARED_REPOS_LIST"; then
+    elif [ "$PROVIDED" = "true" ]; then
         # Trigger the shared hook repository checkout manually
-        cp "$GITHOOKS_CLONE_DIR/base-template-wrapper.sh" ".githooks.shared.trigger" &&
-            chmod +x ".githooks.shared.trigger" &&
-            ./.githooks.shared.trigger
-        rm -f .githooks.shared.trigger
-
+        "$GITHOOKS_CLONE_DIR/cli.sh" shared update --global
```

Comment:
OK, I see now why is `.githooks.shared.trigger` retired.
Just to double-check, brand new installations have this `cli.sh` in place at this point already, right?

Reply:
Correct.